### PR TITLE
INT-17 Enable the WikiaInYourLang extension cross-language

### PR DESCRIPTION
**Pull request by adamkarminski**
This pull request enables all users to see a notification if they are at a wikia that has a sibling one in their native language.

### CODE REVIEW

File: extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js [hunk 1]
```diff
@@ -45,16 +40,22 @@ require(
...
 		function getTargetLanguage() {
 			var browserLanguage = window.navigator.language || window.navigator.userLanguage,
-				geoCountryCode = geo.getCountryCode().toLowerCase();
+				geoCountryCode = geo.getCountryCode().toLowerCase(),
+				targetLanguage;
 
-			// Check if a browser's language is one of the supported languages
-			if (typeof browserLanguage === 'string' && $.inArray(browserLanguage.substr(0, 2), supportedLanguages) !== -1) {
+
+			if (typeof w.wgUserName !== 'null') {
```
Comment (macbre) [inline extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js]: ``typeof null`` is `object`.. Use `wgUserName !== null` instead.

File: extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js [hunk 2]
```diff
@@ -112,7 +113,7 @@ require(
 			$('.global-notification.notify').click(function (event) {
 				if (event.target.parentElement.className.indexOf('close') !== -1) {
 					onNotificationClosed();
-				} else if (event.target.id.indexOf('wikia-in-your-lang-link') !== -1) {
+				} else if (event.target.className.indexOf('text') !== -1) {
```
Comment (macbre) [inline extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js]: Minor one. As you're already using jQuery, it makes sense to use it here as well (to increase the readability of the code:

```js
var $target = $(event.target);

if ($target.hasClass('text') {
  // ..
}
```

Comment (adamkarminski) [inline extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js]: Thank you, it's a life-saver. :)

File: extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js (hunk 1 above)
Comment (adamkarminski) [inline extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js]: True, thanks!

File: extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js [hunk 3]
```diff
@@ -45,16 +40,22 @@ require(
...
 		function getTargetLanguage() {
 			var browserLanguage = window.navigator.language || window.navigator.userLanguage,
-				geoCountryCode = geo.getCountryCode().toLowerCase();
+				geoCountryCode = geo.getCountryCode().toLowerCase(),
+				targetLanguage;
 
-			// Check if a browser's language is one of the supported languages
-			if (typeof browserLanguage === 'string' && $.inArray(browserLanguage.substr(0, 2), supportedLanguages) !== -1) {
+
+			if (w.wgUserName !== 'null') {
```
Comment (macbre) [inline extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js]: ``null`` not ``"null"`` as a string :)

**Pull request merged**